integration/fuse/src/main/java/alluxio/fuse/cli/FuseShell.java

```diff
@@ -71,53 +94,36 @@ public URIStatus runCommand(AlluxioURI uri) throws InvalidArgumentException {
       throw new InvalidArgumentException("Command is needed in Fuse shell");
     }
 
-    String cmdType = cmds[2];
-    switch (CommandType.fromValue(cmdType)) {
-      case METADATA_CACHE:
-        MetadataCacheCommand command = new MetadataCacheCommand(mFileSystem, mConf);
-        return command.run(path, Arrays.copyOfRange(cmds, 3, cmds.length));
-      default:
-        throw new InvalidArgumentException("Fuse shell command not found");
+    FuseCommand command = mCommands.get(cmds[2]);
+    if (command == null) {
+      logUsage();
+      throw new InvalidArgumentException(String.format("%s is an unknown command.", cmds[2]));
     }
+    String [] currArgs = Arrays.copyOfRange(cmds, 2, cmds.length);
+    try {
+      while (command.hasSubCommand()) {
+        if (currArgs.length < 2) {
+          throw new InvalidArgumentException("No sub-command is specified");
+        }
+        if (!command.getSubCommands().containsKey(currArgs[1])) {
+          throw new InvalidArgumentException("Unknown sub-command: " + currArgs[1]);
+        }
+        command = command.getSubCommands().get(currArgs[1]);
+        currArgs = Arrays.copyOfRange(currArgs, 1, currArgs.length);
+      }
+      command.validateArgs(Arrays.copyOfRange(currArgs, 1, currArgs.length));
+    } catch (Exception e) {
+      LOG.info(e.getMessage());
```

Comment:
LOG is different from the console output, and better to well format the message in one LOG.
Like `Command xxxx failed because of xxx. For detailed usage please type XXX`

Reply:
Ok